webserver/host.go

```diff
@@ -67,6 +72,9 @@ func (c Controller) HostResources(res http.ResponseWriter, req *http.Request, pa
 		Memory: memory,
 	}
 
-	json.NewEncoder(res).Encode(&result)
+	err = json.NewEncoder(res).Encode(&result)
+	if err != nil {
+		return errors.Wrap(err, "fail to encode host usage payload")
```

Comment:
This should use a log.WithError since the body might have already been set.

Reply:
Oh yes right